src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSsl.cs

```diff
@@ -288,6 +293,11 @@ internal static bool DoSslHandshake(SafeSslHandle context, byte[] recvBuf, int r
                 {
                     sendCount = BioRead(context.OutputBio, sendBuf, sendCount);
                 }
+                catch (Exception e) when (handshakeException != null)
+                {
+                    // If we already have handshake exception, ignore any exception from BioRead().
+                    handshakeException = e;
```

Comment:
This comment doesn't seem to match with what's being done.  This code is replacing handshakeException with the one thrown from bioRead, not ignoring it.  Am I misunderstanding?

Reply:
I broke it when I did rewrite with exception filer ;( 
fixed now